docs/howto/inspect_suite_database.ipynb

```diff
@@ -0,0 +1,473 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Suite database viewer\n",
+    "This notebook was created to capture common commands which may be useful to developers\n",
+    "working on, or helping users with Cylc Suite databases.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 105,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Set the parameters of the suite whose database you want to investigate\n",
+    "USER = \"jbloggs\"\n",
+    "SUITENAME = \"simplest\"\n",
+    "SUITE_FILE_GLOB_PATTERN = f\"/home/*/{USER}/cylc-run/{SUITENAME}/\"\n",
```

Comment:
This doesn't work for the usual `/home/$USER/` home dir structure.  
Maybe better to have a separate home dir variable, so it's more obvious that that needs customizating, and then automatically add the standard cylc-run path after that?

Reply:
Yes, so I've changed this to using
```python
os.getenv('HOME')
```